Sample-Notebooks/Example - Azure Storage VT Hash Lookup.ipynb

```diff
@@ -0,0 +1,196 @@
+{
```

Comment:
<p>It doesn't matter for our TI lookup (since it has a LRU cache) but you could collect the hashes into a set to avoid duplicates</p><p>Also you can use lookup_iocs(data=alertout, obs_col="Md5Hash") with a DF input directly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/78/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Applied this suggested change, need to break the habit of using loops with dataframes.